chore(cloud-rad): Add code fencing

fhinkel opened the pull request:
## Code fencing
Code examples need code fencing around them to distingish from
rich text for TSDoc.

Internally b/179483748

Script used:
https://github.com/fhinkel/cloud-rad-script/blob/main/fixExampleComments.js

JustinBeckwith reviewed (changes requested):
This appears to be adding double lines of backticks everywhere

fhinkel committed:
chore(cloud-rad): Add code fencing

Code examples need code fencing around them to distingish from
rich text for TSDoc.

Internally b/179483748

Script used:
https://github.com/fhinkel/cloud-rad-script/blob/main/fixExampleComments.js

fhinkel commented:
😱 Must have run a script a second time. Fixed.
